upload-cloud-assembly/action.yml

```diff
@@ -26,16 +26,6 @@ runs:
         INPUT_AWS_S3_BUCKET_NAME: ${{ inputs.aws-s3-bucket-name }}
         INPUT_CDK_APP_DIR: ${{ inputs.cdk-app-dir }}
       run: |
-        cdk_assembly_dir="${INPUT_CDK_ASSEMBLY_DIR:-$GITHUB_WORKSPACE}"
-        cd "$cdk_assembly_dir"
-        tmp_dir="/tmp/${GITHUB_ACTION:-$(date -u +%s)}"
-        mkdir -p "$tmp_dir"
-        cloud_assembly_archive="$tmp_dir/cloud-assembly.zip"
-        if [ ! -f "cdk.json" ]; then
-          echo "No cdk.json file found in '$input_cdk_assembly_path'. You may need to set the 'cdk-app-dir' input."
-          exit 1
-        fi
-
         bash "$GITHUB_ACTION_PATH/action.sh" \
           --aws-s3-bucket-name "$INPUT_AWS_S3_BUCKET_NAME" \
-          --cloud-assembly-archive "$cloud_assembly_archive"
+          --cdk-app-dir "${INPUT_CDK_APP_DIR:-$GITHUB_WORKSPACE}"
```

Comment:
nit: As a general pattern I find the readability to be better if we perform these type of operations before the invocation of the shell script. Makes it easier to add comments, explanations etc. if needed, as well as doing more complex operations. E.g.,
```shell
        cdk_app_dir="${INPUT_CDK_APP_DIR:-$GITHUB_WORKSPACE}"
        bash "$GITHUB_ACTION_PATH/action.sh" \
          --aws-s3-bucket-name "$INPUT_AWS_S3_BUCKET_NAME" \
          --cdk-app-dir "$cdk_app_dir"
```

Reply:
https://github.com/capralifecycle/actions-lib/pull/6/commits/0bc81b019e2e015d89f72341f1eb430a4d5631a1